### PR TITLE
Add new sentence

### PR DESCRIPTION
**Pull request by Fakhatova**

### CODE REVIEW

**Comment by tylrs** [comment]
Mm I don't know about that

File: README.md [hunk 1]
```diff
@@ -1 +1,2 @@
 # Hello
```
Comment (tylrs) [inline README.md:1]: I think we need something else here

File: README.md [hunk 2]
```diff
@@ -1 +1,2 @@
...
+### Hello Again
```
Comment (tylrs) [inline README.md:2]: Maybe bold here

Comment (Fakhatova) [reply README.md:2]: yeah sure, can you change on your new branch

**Pull request merged**